__tests__/cli/deploy/bundle/DeployBundle.handler.test.ts

```diff
@@ -679,4 +679,118 @@ describe("bundle Handler", () => {
         expect(err).toBeInstanceOf(ImperativeError);
         expect(err.message).toContain("--timeout parameter is too small");
     });
+    it("should complain with no cicshlq parameter", async () => {
+
+        let parms: IHandlerParameters;
```

Comment:
Suggest setting up common parms in a beforeAll function to avoid duplicating in all of these tests. You could put it inside a nested `describe` for just these tests.
In fact the whole of the test could extracted into a function which you just pass in the value of cicshlq into

Reply:
I'm not sure that beforeAll would help... the parms are too different from one test to the next. But I guess some utility methods could be added where three or four tests need essentially the same values.

Reply:
sorry I was only referring to the new tests for --cicshlq which seem to be identical except for the value of the parameter itself, this seems like a good case to extract that test into a function and parameterise it with the value.